src/data/dialControlParmeters.tsx

```diff
@@ -0,0 +1,106 @@
+import IDialControlParameters from "../interfaces/IDailControlParameters"
+import getDefaultContextWrapperValues from "../hooks/getDefaultContextWrapperValues";
+
+const defaultValues = getDefaultContextWrapperValues();
+
+function createParametersObject(
+    min: number,
+    max: number,
+    title: string,
+    localStorageKey: string,
+    defaultValue : number,
+    factor : number = 1
+     ) : IDialControlParameters{
+         return {
+             min,
+             max,
+             title, localStorageKey,
+             defaultValue,
+             factor
+         }
+}
+
+export const envelopeAttackParameters = createParametersObject(
+    1,
+    2000,
+    "ATTACK",
+    "envelopeAttackMs",
+    defaultValues.envelopeParameters.decayMs,
+)
+
+export const envelopeDecayParameters = createParametersObject(
+    1,
+    2000,
+    "DECAY",
+    "envelopeDecayMs",
+    defaultValues.envelopeParameters.decayMs,
+) 
+
+export const envelopeSustainParameters  = createParametersObject(
+    0,
+    1000,
+    "SUSTAIN",
+    "envelopeSustainMs",
+    defaultValues.envelopeParameters.sustain
+)
+
+export const envelopeReleaseParameters  = createParametersObject(
+    0,
+    2000,
+    "RELEASE",
+    "envelopeReleaseMs",
+    defaultValues.envelopeParameters.releaseMs
+)
+
+export const filterFrequencyParameters = createParametersObject(
+    40,
+    20000,
+    "FREQUENCY",
+    "filterFrequency",
+    defaultValues.filterParameters.freq
+)
+
+export const filterQParameters = createParametersObject(
+    1,
+    20000,
```

Comment:
Q should go from 0.01 to 20 (these were the previous bounds too)

Reply:
I haven't figured out how to get the knob to play nice with non-integers